syft/pkg/cataloger/config.go

```diff
@@ -8,23 +8,14 @@ import (
 )
 
 // TODO: these field naming vs helper function naming schemes are inconsistent.
-
 type Config struct {
-	Search      SearchConfig
-	Golang      golang.GoCatalogerOpts
-	LinuxKernel kernel.LinuxCatalogerConfig
-	Python      python.CatalogerConfig
-	Catalogers  []string
-	Parallelism int
-}
-
-func DefaultConfig() Config {
```

Comment:
why delete the `DefaultConfig` method?

Reply:
This function was only used as a part of `*_test.go` files. It was moved here:https://github.com/anchore/syft/blob/c7272fd6a5417f4181f72293a169e394013e5165/test/integration/utils_test.go#L57-L65

Apologies for the boy scout change on an unrelated PR - my IDE was yelling about this being `deadcode` and I could not figure out why - the refactor over to test resolved that issue